io.elementary.Sdk.json.in

```diff
@@ -27,8 +27,8 @@
         "--env=ALSA_CONFIG_PATH=/usr/share/alsa/alsa-flatpak.conf",
         "--env=GI_TYPELIB_PATH=/app/lib/girepository-1.0",
         "--env=XDG_DATA_DIRS=/app/share:/usr/share:/usr/share/runtime/share:/run/host/share",
-        "--sdk=io.elementary.Sdk//daily",
-        "--runtime=io.elementary.Platform//daily"
+        "--sdk=io.elementary.Sdk//@branch@",
+        "--runtime=io.elementary.Platform//@branch@"
```

Comment:
I think it would be better to include the architecture here too. I regularly have the `aarch64` and `x86_64` SDKs installed for cross compilation, so it would be good to prevent uncertainty wherever possible.

Meson's `target_machine.cpu ()` built-in will get what you need.

Reply:
it is actually automatically added